app/src/main/java/com/example/musictoned/Navigation.kt

```diff
@@ -93,6 +93,9 @@ fun MusicTonedNavHost(navController: NavHostController = rememberNavController()
                 },
                 onNavigateToSettings = {
                     navController.navigate(SETTINGS_ROUTE)
+                },
+                onNavigateToSpotifyBeta = {
```

Comment:
I removed the Spotify beta stuff since it was just placeholders for Harsh. Probably don't need this anymore